airbyte-integrations/connectors/source-okta/source_okta/config_migration.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import logging
+from typing import Any, List, Mapping
+
+from airbyte_cdk.config_observation import create_connector_config_control_message
+from airbyte_cdk.entrypoint import AirbyteEntrypoint
+from airbyte_cdk.sources import Source
+from airbyte_cdk.sources.message import InMemoryMessageRepository, MessageRepository
+
+logger = logging.getLogger("airbyte")
+
+
+class OktaConfigMigration:
```

Comment:
Please add unit tests for this migration

Reply:
Added unit tests for config migration